readme.md

```diff
@@ -35,6 +35,12 @@ camelCase('Foo-Bar', {pascalCase: true});
 camelCase('--foo.bar', {pascalCase: false});
 //=> 'fooBar'
 
+camelCase('Foo-Bar', {preserveConsecutiveUppercase: true});
```

Comment:
The `bar` here doesn't have consecutive uppercase characters though.

Reply:
Ah right. Fixed it.

Reply:
You need to update the type docs too.

Reply:
Right. Just updated.